### ISSUE TITLE
Multiline comments not working in deno doc

### ISSUE THREAD

**Issue by humancalico** [denoland/deno#4755]
Running `deno doc` on 
```ts
/**
* Hello there, this is a multiline JSdoc.
*
* It has many lines
*
* Or not that many?
*/
export function foo(a: string, b?: number, cb: (...cbArgs: unknown[]) => void, ...args: unknown[]): void {
    console.log("Hello world");
}
```
gives
```
function foo(a: string, b?: number, cb: (cbArgs: unknown[]) => void, args: unknown[]): void
  Hello there, this is a multiline JSdoc.

```
instead of
```

function foo(a: string, b?: number, cb: (cbArgs: unknown[]) => void, args: unknown[]): void
  Hello there, this is a multiline JSdoc.

  It has many lines

  Or not that many?
```

----
Tests for this do exist. [here](https://github.com/denoland/deno/blob/a7a3a771a0a0cb2df2a14bcf4b8c38bc69525dd6/cli/doc/tests.rs#L47-L151)

**Comment by ry** [comment]
Thanks @humancalico - I would also expect the full docs to be printed in the CLI, but I expect the intention was to just print an index of docs, and allow the user to drill into further by doing `deno doc foo.js foo`.  Is that correct @lucacasonato ?

That said, I think it's a bit confusing and limiting to not just display the complete documentation up front.

**Comment by humancalico** [comment]
@ry I guess adding `--full/--complete` flag for `deno doc` for printing the full docs would make much more sense.
or the other way around it could be for `deno doc` to print the full doc and adding an `--index` flag for just printing the index

**Comment by lucacasonato** [comment]
This was done to save on space in the overview view. If I remember correctly go doc does it this way too (it even truncates function signatures). All lines should be displayed in the details view. The first line of JSDoc is also what is displayed by VS code autocomplete, so this is nothing new in JS land. Most people won’t need examples for how to use something when they are just browsing through the package’s exports. They get examples when they look at specific functions in the details view.

This might be made more clear by adding `[...]` after any truncated JSDoc.

I think a `—no-truncate` option would also work.

**Comment by rajeev-kumar-sharma** [comment]
I would like to work on this bug.
I will implement it all possible ways suggest there and we will finalise later.

May you direct me, where to start.

**Comment by bartlomieju** [comment]
Fixed in #6541
